packages/pwa/pwa-kit-config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "app": {
+        "url": {
+            "locale": "path",
+            "siteAlias": "none"
```

Comment:
Here is a question for you. Is `siteAlias` something that is going to be mandatory? My initial thought about alias's is that they are almost always optional. E.g. it might be perfectly fine for someone to want to use the actual site name and not have an alias for it, like `http://www.bensbarginbin.com/refarch/categorys/cheap-stuff`. 

So this begs the question, is the proper config option really "siteId"? 

Since all the siteid/sitealias stuff is out of scope for this ticket, you can also opt to no do site alias stuff as all in this ticket.

Reply:
I am thinking a bit a head for the multisite here, because we kinda know from the multisite spike, it is more likely the siteAlias will be used more than the siteId itself. 

The reason I put the siteAlias because I want to bring up a convo around the way we write the `buildPathWithUrlConfig` function [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/228/files#diff-f46ab45bbea466e88ac5ab4c430662f8f265bce3754178d40040643bdfc72b3aR244).  

With regard to our previous conversation on the spike around avoiding iterating through the url config object because they are probably no literal mapping, in this func, I am picking out the `locale` and `siteaAlias` because I know they will be in the config file. The function does not look really dynamic and smart so to speak. 
This would work for those two, but if another param comes up, let's say the customer wants another param to their config, will they have to do some changes to this function as well?

Is there a better way to write this function a bit more dynamic?

Reply:
I think we skipped over the comment about choosing `siteAlias` over `siteId` here. What I was trying to convey is that `aliasing` is in itself a feature, if we are going to be supporting aliasing for the site it's not a crazy idea that we support it for the locale as well.. so you might want to think about that and how it effects the names of your config options, it looks like you are saying that for locale it's always going to be the literal value, and for the site it's going to be an alias that we make, it's  not very consistent